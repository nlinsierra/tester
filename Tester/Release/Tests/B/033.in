5D 4C TD 6S 7C 3S AD 8H 7D QS 7S 7H 4D 2C 9C 9S 5C 4H 5H JD 8C 2H QH 2D TC
