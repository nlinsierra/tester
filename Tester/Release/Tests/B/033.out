27
3C 6C JC QC KC AC 3D 6D 8D 9D QD KD 3H 6H 9H TH JH KH AH 2S 4S 5S 8S TS JS KS AS
